e2e/utils/command-utils.ts

```diff
@@ -218,14 +226,14 @@ export function runCommandUntil(
       CI: 'true',
       ...getStrippedEnvironmentVariables(),
       FORCE_COLOR: 'false',
-    },
+    } as any,
   });
   return new Promise((res, rej) => {
     let output = '';
     let complete = false;
 
     function checkCriteria(c) {
-      output += c.toString();
+      output += c?.toString();
```

Comment:
No need for this. The `checkCriteria` is called on `data`, so there will always be an input.